x11-themes/papirus-icon-theme/papirus-icon-theme-20210401-r1.ebuild

```diff
@@ -7,11 +7,17 @@ inherit xdg
 
 DESCRIPTION="Free and open source SVG icon theme"
 HOMEPAGE="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme"
-SRC_URI="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
```

Comment:
Why revbump for this?

Reply:
Doesn't this require a revbump? (I did to be sure)

Reply:
removed revbump

Reply:
[generally](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html), only if it is risky or changes installed files